app/models/course_user.rb

```diff
@@ -72,6 +72,7 @@ class CourseUser < ActiveRecord::Base
   scope :instructors, -> { staff }
   scope :students, -> { where(role: roles[:student]) }
 
+  scope :not_phantom, -> { where(phantom: false) }
```

Comment:
I prefer positive nouns, mmm. We need to think of other names. Negations can be confusing when you have many scopes linked together.